docs/AOT.md

```diff
@@ -163,7 +163,7 @@ The makefile provides the following targets:
 * `download_weights`: it downloads the Resnet50 network model in the Caffe2 format.
 * `build/resnet50.o`: it generates the bundle files using the Glow image-classifier as described above.
   The concrete command line looks like this:
-  `image-classifier tests/images/imagenet/cat_285.png -image_mode=0to1 -m resnet50 -cpu -emit-bundle build`
+  `image-classifier tests/images/imagenet/cat_285.png -image_mode=0to1 -m resnet50 -model_expected_input_name=gpu_0/data -cpu -emit-bundle build`
```

Comment:
`model_expected_input_name` loos a bit long. Also, I don't think it's an expected name, but this is the actual name.
I think that `input_name` or even `input` would be enough. Regarding the quotes around the name, my preference is to not have them, like I believe it's mode common to do. The only advantage of having the quote would be to enable the usage of space in names. It is a use case we want to cover ?

Reply:
Yeah I will remove `expected`. For `model` -- I wanted it to be very clear it has nothing to do with the file name of the image input being classified. I changed it to `--model_input_name` -- it's at least a little shorter.